pac4j-saml/src/main/java/org/pac4j/saml/client/SAML2Client.java

```diff
@@ -150,7 +150,7 @@ protected void initSAMLContextProvider(final MetadataResolver metadataManager) {
     }
 
     protected MetadataResolver initServiceProviderMetadataResolver(final WebContext context) {
-        this.spMetadataResolver = new SAML2ServiceProviderMetadataResolver(this.configuration.getServiceProviderMetadataPath(),
+		this.spMetadataResolver = new SAML2ServiceProviderMetadataResolver(this.configuration.getServiceProviderMetadataPath(),
```

Comment:
I think that passing the `configuration` could be sufficient, couldn't it? To mirror what's done for the identity provider...